src/components/reformed.js

```diff
@@ -89,10 +91,10 @@ const makeWrapper = WrappedComponent => {
       }
     }
 
-    bindInput(name) {
+    bindInput(name, isFile = false) {
```

Comment:
Jeg testet forslaget ditt nå. Når du velger en fil vil`bindInput(name,  undefined)` trigges og `this.state.model[name]` får fil informasjonen. Dvs at `value` aldri blir satt som `undefined`.